db/migrate/20200901074007_add_unique_index_to_articles_slug_and_feed_source_url.rb

```diff
@@ -0,0 +1,33 @@
+class AddUniqueIndexToArticlesSlugAndFeedSourceUrl < ActiveRecord::Migration[6.0]
```

Comment:
as there's a tiny chance that a migration breaks but one of the indexes passes through we better be idempotent as much as possible here ;)